src/config/containers-and-vms/chroot.md

```diff
@@ -0,0 +1,80 @@
+# Creating and using chroots
+
+Similar to container solutions like docker or podman, chroots can be set up and
+used for many purposes, including:
+
+- running glibc software on musl (and vice versa)
+- running software in a more controlled or sandboxed environment
+- creating a rootfs for bootstrapping a system
+
+## Creation
+
+Create a directory that will contain the chroot, and install a base system in it
+via the `base-voidstrap` package:
+
+```
+$ mkdir <chroot_dir>
+$ XBPS_TARGET_ARCH=<chroot_arch> xbps-install -M -r <chroot_dir> -R <repository> base-voidstrap
```

Comment:
if we already use xchroot below, is it worth mentioning `xvoidstrap` here as well?